packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js

```diff
@@ -0,0 +1,55 @@
+const _ =  require('underscore');
+const loginEnBundle = require('../../../i18n/src/json/login.json');
```

Comment:
Because we're parsing the `json` file, line numbers in which the error occurs are going to be off. To improve the reporting of this rule, we should use the `properties` processor in [okta-ui-configs](https://github.com/okta/okta-ui-configs/blob/master/packages/eslint-plugin-okta/docs/processors/properties.md).

For now, can we make sure the `line` and `column` properties are set to `0`?

Reply:
Also - in the long term our lint script shouldn't be dependent on `propertiesToJSON` executing successfully. Instead, we should treat the `*.properties` file as our source of truth and import it here.

Reply:
I feel like the current error is already pointing out the i18n key which should be easy to find in the json file (and in most cases there should only be one). But, with that being said I will look into setting the line and column to 0 for this PR